src/components/Footer/index.jsx

```diff
@@ -63,7 +63,7 @@ class Footer extends React.Component {
                   </a>
                 </li>
                 <li className="nav-item">
-                  <Link className="nav-link" to={enterpriseSlug ? `/${enterpriseSlug}/support` : '/public/support'}>
+                  <Link className="nav-link" to={enterpriseSlug ? `/${enterpriseSlug}/admin/support` : '/public/support'}>
```

Comment:
Seems reasonable, but just confirming that we want to move this url?

Reply:
Yeah, I think we're OK to move this route. All the other pages nested under the /<enterprise-slug>/ route also have the `admin` prefix like @anaidu0 is changing this route to 👍